### PR TITLE
Dutch language improvements (also legal)

### PR DESCRIPTION
**Pull request by noud**
hi Laravel Daily, PovilasKorop  and others,

i did add to the readme how i do my translation.

i did improve the dutch language translation.

i did add translation of custom_fields in the view.

I hope this all is welcome.

In the actual dutch translation file some wordings are improved.
In the Netherlands there are more legal requirements to an invoice.
Like kvk no. being Chamber of Commerce no.

Noud

### CODE REVIEW

**Comment by mc0de** [comment]
@noud thank you, will look into this later this week

**Comment by noud** [comment]
David,
i am adding some more.
and i will check the existing examples now.

**Comment by noud** [comment]
@mc0de i did scale down the PR. so easier to pull-up. this should be it for the moment.

now it is just:
- better wording in the dutch language file
- translate custom_fields if translation available else ucfirst.
- do not do amount_in_words if isLocale in not en.

i did create a small laravel-invoices-usage project for immediate testing
w/ just the controllers for example usage and there routes.
https://github.com/noud/laravel-invoices-usage

Is this pull-request now workable like this?

thanks,
Noud

**Comment by noud** [comment]
can we still have this merged? i trimmed down to 2 files.

File: resources/views/templates/default.blade.php [hunk 1]
```diff
@@ -100,7 +100,11 @@
 
                         @foreach($invoice->seller->custom_fields as $key => $value)
                             <p class="seller-custom-field">
-                                {{ ucfirst($key) }}: {{ $value }}
+                                @if('invoices::invoice.' . $key === __('invoices::invoice.' . $key))
```
Comment (mc0de) [inline resources/views/templates/default.blade.php:103]: i don't see how this condition ever is going to be true

File: resources/views/templates/default.blade.php [hunk 2]
```diff
@@ -138,7 +142,11 @@
 
                         @foreach($invoice->buyer->custom_fields as $key => $value)
                             <p class="buyer-custom-field">
-                                {{ ucfirst($key) }}: {{ $value }}
+                                @if('invoices::invoice.' . $key === __('invoices::invoice.' . $key))
```
Comment (mc0de) [inline resources/views/templates/default.blade.php:145]: i don't see how this condition ever is going to be true

File: resources/views/templates/default.blade.php [hunk 3]
```diff
@@ -255,9 +263,11 @@
             </p>
         @endif
 
-        <p>
-            {{ trans('invoices::invoice.amount_in_words') }}: {{ $invoice->getTotalAmountInWords() }}
-        </p>
+        @if(App::isLocale('en'))
```
Comment (mc0de) [inline resources/views/templates/default.blade.php:266]: if you need this specific logic please make your own template, this is why we support templating and invoice isn't necessarily same locale as your application

https://github.com/LaravelDaily/laravel-invoices/issues/30

**Review by mc0de — COMMENTED** [review]
In addition to my previous comments the only file I can accept is `resources/lang/nl/invoice.php `

The other problem is that your commit history isn't clean and commit messages are with mistakes.

Please make new pull request only with changes for `resources/lang/nl/invoice.php `

**Pull request closed without merging**